jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/popup/TasksReassignmentViewImpl.html

```diff
@@ -0,0 +1,24 @@
+<div>
+    <div>
+        <div class="form-horizontal">
+            <fieldset>
+
+                <div class="col-md-12" style="margin-bottom: 10px;">
+                    <span data-i18n-key="To_Reassign_Selected_Tasks_Introduce_UserId"></span>
```

Comment:
I wonder if we should present this string as is, or simply append it to a info icon next to user label.

Reply:
Changed the help text to a popover on the info icon...